packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -243,6 +245,15 @@ class MediaQueryData {
   ///  * [Window.AccessibilityFeatures], where the setting originates.
   final bool invertColors;
 
+  /// Whether the user requested a high contrast between foreground and background
+  /// content, in either:
+  /// * Settings -> Accessibility -> Increase Contrast (iOS)
+  /// * Settings -> Accessibility -> High contrast text (Android)
+  ///
+  /// This flag is currently only updated on iOS devices that are running iOS 13
+  /// or above, and Android devices that are running Android Q and above.
```

Comment:
I think we decided that, for now, we're not reporting it for Android Q because it's still an experimental Android feature.